lib/test_runner.rb

```diff
@@ -4,6 +4,12 @@
 
 class TestRunner < Mumukit::Stub
   def run_compilation!(test_definition)
-    Gobstones::Spec::Runner.new(Gobstones::Spec::Language::Gobstones).run!(test_definition)
+    result =  Gobstones::Spec::Runner.new(Gobstones::Spec::Language::Gobstones).run!(test_definition)
+
+    if result[1] == :failed
+      result[0] = "<pre>#{result[0]}</pre>"
+    end
```

Comment:
The problem with this is that we are not considering failed html outputs returned by the runner - remember that the runner may fail with an html output of the boards.

This pre behaviour should be implemented within the test-runner gem
The problem with this is that we are not considering failed html outputs returned by the runner - remember that the runner may fail with an html output of the boards.

This pre behaviour should be implemented within the test-runner gem, I think